pkg/volume/azure_file/azure_provision.go

```diff
@@ -36,37 +36,44 @@ import (
 	utilstrings "k8s.io/utils/strings"
 )
 
-var _ volume.DeletableVolumePlugin = &azureFilePlugin{}
-var _ volume.ProvisionableVolumePlugin = &azureFilePlugin{}
+var (
+	_ volume.DeletableVolumePlugin     = &azureFilePlugin{}
+	_ volume.ProvisionableVolumePlugin = &azureFilePlugin{}
+
+	resourceGroupAnnotation = "resourceGroup"
```

Comment:
use as `resource-group`, maybe more matching to naming rule as `azure-file-dynamic-provisioner`
```
Annotations: map[string]string{
				util.VolumeDynamicallyCreatedByKey: "azure-file-dynamic-provisioner",					util.VolumeDynamicallyCreatedByKey: "azure-file-dynamic-provisioner"
```

Reply:
naming like this `kubernetes.io/azure-file-resource-group` ?

Reply:
done